packages/devextreme/js/ui/hierarchical_collection/ui.hierarchical_collection_widget.js

```diff
@@ -105,7 +105,17 @@ const HierarchicalCollectionWidget = CollectionWidget.inherit({
     },
 
     _getIconContainer: function(itemData) {
-        return itemData.icon ? getImageContainer(itemData.icon) : undefined;
+        if(!itemData.icon) {
+            return undefined;
+        }
+
+        const $imageContainer = getImageContainer(itemData.icon);
+
+        if($imageContainer.is('img')) {
+            $imageContainer.attr('alt', itemData.text ?? `${this.NAME} item icon`);
```

Comment:
I think we should use messageLocalization.format() here.

Reply:
I think it's too much here

Reply:
it's supposed to be super rare case when we use img icon + do not use text